oauthproxy.go

```diff
@@ -571,22 +571,37 @@ func (p *OAuthProxy) ManualSignIn(req *http.Request) (string, bool) {
 func (p *OAuthProxy) GetRedirect(req *http.Request) (redirect string, err error) {
 	err = req.ParseForm()
 	if err != nil {
-		return
+		return redirect, err
```

Comment:
I'd prefer to move away from named returns, so best to be explicit here. Normally an error would want an empty string returned
```suggestion
		return "", err
```